frontend/src/services/index.js

```diff
@@ -19,12 +33,19 @@ if (localStorage.getItem("user")) {
 	API.interceptors.response.use(
 		async (config) => {
 			config.headers.Authorization = `Bearer ${accessToken}`;
+		if (!isExpired) {
+		return config;
+		} else {
+			config.headers.Authorization = `Bearer ${localStorage.getItem(
+				"newToken"
+			)}`;
```

Comment:
Same here it should be put in a separate function

Reply:
Discussed with @BrahimS : he think that it should not be put in a separated function. We should not loose too much time on this so I'll validate the PR.